src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const totalCosts = days * 40;
+
+  if (days >= 7) {
+    return totalCosts - 50;
+  }
+
+  if (days >= 3) {
+    return totalCosts - 20;
```

Comment:
Consider defining a constant for the discount value (20) for rentals of 3 or more days to improve code readability and maintainability.